src/extension/package.json

```diff
@@ -62,6 +62,10 @@
           "type": "boolean",
           "default": false,
           "description": "Enable upcoming features; Warning: may cause stability issues."
+        },
+        "wts.legalResources": {
+          "type": "null",
```

Comment:
just curious, why the type is null?

Reply:
So for the type there is `null`, `array`, `number`, `object`, `string`, `boolean`, `object` according to VSCode autocomplete. Some of them show a checkmark like the boolean, some of them show an input field like the number and string. Rest of them show the `Edit in settings.json` like in screenshot above.

Reply:
Also the docs are really limited and I haven't found an extension setting where they only show the strings yet